doctr/models/_utils.py

```diff
@@ -103,10 +105,56 @@ def rotate_page(
     """
     if abs(angle) < min_angle or abs(angle) > 90 - min_angle:
         return image
+    elif abs(angle) > 45:
+        # if image is closer to portrait mode, go for a full 90° rotation
+        image = cv2.rotate(image, cv2.cv2.ROTATE_90_CLOCKWISE)
+        angle += 90  # track the rotation by updating angle
```

Comment:
I might be missing something but: if the angle is higher than 45, and we perform first-hand a 90° rotation, then the remaining rotation to perform is `angle - 90` and not `angle + 90` ?

Reply:
In `cv2.getRotationMatrix2D`, positive angle values mean counter-clockwise rotation. Here I assume the image needs clockwise rotation.